cypress/support/step_definitions/constraint-creation.js

```diff
@@ -100,3 +100,13 @@ when("I use negated pedigree constraint {string} with concept {string} and negat
   cy.get('.ui-inputswitch-slider').eq(2).click();
   cy.get('.update-btn').eq(0).click();
 });
+
+when("I select diagnosis that are part of study {string}", (studyName) => {
```

Comment:
singular: diagnosis, plural: diagnoses,
or you meant it as a mass noun, in which case, the aux verb would be 'is'

Reply:
thanks, I meant diagnoses, it is changed now

Reply:
still the same?